lib/src/collection/bimap.dart

```diff
@@ -151,7 +151,7 @@ class HashBiMap<K, V> implements BiMap<K, V> {
   @override
   void updateAll(V Function(K key, V value) update) {
     for (final key in keys) {
-      _add(key, update(key, _map[key]!), true);
+      _add(key, update(key, _map[key] as V), true);
```

Comment:
handles the nullable `V` case correctly!

This could also iterate over `entries` to avoid the `as` check...